cmake/wasi-sdk-sysroot.cmake

```diff
@@ -124,8 +127,17 @@ function(define_wasi_libc target)
         NM=${CMAKE_NM}
         SYSROOT=${wasi_sysroot}
         EXTRA_CFLAGS=${extra_cflags}
-	TARGET_TRIPLE=${target}
+        TARGET_TRIPLE=${target}
         ${extra_make_flags}
+    COMMAND
+      ${MAKE} -j8 -C ${build_dir}
+        CC=${CMAKE_C_COMPILER}
+        AR=${CMAKE_AR}
+        NM=${CMAKE_NM}
+        SYSROOT=${wasi_sysroot}
+        EXTRA_CFLAGS=${extra_cflags}
+        TARGET_TRIPLE=${target}
+        ${extra_make_flags_lto}
```

Comment:
Could this be split into a separate build rule with a separate build directory? I think it'd be reasonable to add an option to disable/enable the LTO builds and having it bundled in here will make it difficult to separate out. For debugging it would also be useful to have a single build target for just LTO or just without LTO.

Reply:
do you mean to make 4 more copies of wasi-libc source tree?

Reply:
Until wasi-libc supports out-of-tree builds yes I think that will be required.

Reply:
done